### PR TITLE
Sublist

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/sublist

Instructions
Given any two lists A and B, determine if:

- List A is equal to list B; or
- List A contains list B (A is a superlist of B); or
- List A is contained by list B (A is a sublist of B); or
- None of the above is true, thus lists A and B are unequal

Specifically, list A is equal to list B if both lists have the same values in the same order. List A is a superlist of B if A contains a sub-sequence of values equal to B. List A is a sublist of B if B contains a sub-sequence of values equal to A.

### CODE REVIEW

File: sublist/sublist_task.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def convert_int_list_to_str(integers_list):
+    result = ''
+    for integer in integers_list:
+        result += str(integer) + " "
+    return result
+
+
+def check_if_all_items_in_full_list(full_list, sub_list):
```
Comment (AnaKovaGH) [inline sublist/sublist_task.py]: Too long function name

File: sublist/sublist_task.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Task from https://exercism.org/tracks/python/exercises/sublist
+
+SUBLIST = 1
+SUPERLIST = 2
+EQUAL = 3
+UNEQUAL = 4
```
Comment (AnaKovaGH) [inline sublist/sublist_task.py]: Where do you use this constants?

File: sublist/sublist_task.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        return 2
+    else:
+        return 4
+
+
+def check_if_sublist(full_list, sub_list):
+    sub_list_as_str = convert_int_list_to_str(sub_list)
+    full_list_as_str = convert_int_list_to_str(full_list)
+
+    return sub_list_as_str in full_list_as_str and check_if_all_items_in_full_list(full_list, sub_list)
```
Comment (AnaKovaGH) [inline sublist/sublist_task.py]: It is too hard to understand, please re-write it. How it works?

**Review by AnaKovaGH — DISMISSED** [review]

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was commented.

**Comment by annaoliinyk** [comment]
Re-wrote the solution

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please check how to work with sublist(["a", "d", "a", "s"], ["a", "a", "a"])

**Comment by annaoliinyk** [comment]
This was fixed

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**